conferences/2019/javascript.json

```diff
@@ -1055,6 +1055,16 @@
     "cfpUrl": "https://2019.jsconfkorea.com/en/call-for-proposals",
     "cfpEndDate": "2019-04-26"
   },
+  {
+    "name": "RxJS Live",
+    "url": "https://www.rxjs.live/",
+    "startDate": "2019-09-05",
+    "endDate": "2019-09-06",
+    "city": "Las Vegas, CA",
+    "country": "USA",
```

Comment:
Tests are failing because this should be `U.S.A.` for consistency. :)

Reply:
added the fix

Reply:
👌

Reply:
thanks!